pkg/policycache/cache.go

```diff
@@ -27,17 +27,23 @@ type pMap struct {
 type policyCache struct {
 	pMap
 	logr.Logger
+	// list/get cluster policy resource
+	pLister kyvernolister.ClusterPolicyLister
+
+	// npLister can list/get namespace policy from the shared informer's store
+	npLister kyvernolister.PolicyLister
 }
 
 // Interface ...
 type Interface interface {
 	Add(policy *kyverno.ClusterPolicy)
 	Remove(policy *kyverno.ClusterPolicy)
-	Get(pkey PolicyType, nspace *string) []*kyverno.ClusterPolicy
+	Get(pkey PolicyType, kind *string, nspace *string) []string
```

Comment:
Do we need this? I don't see it is used anywhere.

Reply:
We used in `cache_test.go` for length of policies name.